index.html

```diff
@@ -989,53 +1005,67 @@ <h2>Sharing theme</h2>
           return navigator?.share || navigator?.clipboard;
         },
         share() {
-          const title = "Spellie";
-          const wordsMatch = [
-            [
-              { state: "miss" },
-              { state: "present" },
-              { state: "match" },
-              { state: "miss" },
-            ],
-          ];
-          const wordsEmojis = guessesAsEmojis(
-            wordsMatch,
-            this.settings.sharingEmojis
-          );
-          let text = `ⓦⓞⓡⓓ\n${wordsEmojis}\n🤔🤔🤔🤔\nⓗⓔⓡⓞ🦸`;
-          let shareType = "basic";
+          let title = "Spellie";
+          let text;
+          let shareType;
           if ((this.solved || this.failed) && this.page === "") {
-            const nGuesses = this.failed ? "-" : this.currentGuessIndex;
-            text =
-              `${this.level}: ${nGuesses}/${maxGuesses}` +
-              "\n" +
-              guessesAsEmojis(this.guesses, this.settings.sharingEmojis);
+            // Spellie #10 (medium)
+            // 🕸️🕸️🕸️🕸️🕸️
+            // 🕸️🕸️🌻🍀🕸️
+            // 🍀🍀🕸️🍀🕸️
+            // 🍀🍀🍀🍀🍀
+            title = `Spellie #${this.puzzleId} (${this.settings.difficulty})`;
+            text = guessesAsEmojis(this.guesses, this.settings.sharingEmojis);
             shareType = "puzzle";
           } else if (
             this.page === "#collection" &&
             this.unlockedEmojis.length === 1
           ) {
+            // Spellie
+            // I collected my first emoji! 👩‍🦱
             text = `I collected my first emoji! ${this.unlockedEmojis[0].emoji}`;
             shareType = "emojis";
           } else if (
             this.page === "#collection" &&
             this.unlockedEmojis.length > 1
           ) {
+            // Spellie
+            // I've collected 10 emojis:
+            // 🐈🐩5️⃣4️⃣👧🌳🖌️♟️🧗‍♀️👩‍🦱
             text = `I've collected ${
               this.unlockedEmojis.length
             } emojis:\n${this.unlockedEmojis.map((e) => e.emoji).join("")}`;
             shareType = "emojis";
+          } else {
+            // Spellie
+            // ⓦⓞⓡⓓ
+            // 🕸️🌻🍀🕸️
+            // 🤔🤔🤔🤔
+            // ⓗⓔⓡⓞ🦸
+            const wordsMatch = [
+              [
+                { state: "miss" },
+                { state: "present" },
+                { state: "match" },
+                { state: "miss" },
+              ],
+            ];
+            const wordsEmojis = guessesAsEmojis(
+              wordsMatch,
+              this.settings.sharingEmojis
+            );
+            text = `ⓦⓞⓡⓓ\n${wordsEmojis}\n🤔🤔🤔🤔\nⓗⓔⓡⓞ🦸`;
+            shareType = "basic";
           }
-          const url = window.location.origin;
+
           const shareData = {
             title,
             text: `${text}\n`,
-            url,
```

Comment:
Do we still want the url for basic sharing?

But if we drop the url from shareData, I don't think we need the trailing `\n` anymore. Although maybe safer to leave it there.

Reply:
I guess wordle doesn't have the url, right? Let's try without